hummingbot/connector/exchange/ascend_ex/ascend_ex_exchange.py

```diff
@@ -992,7 +1017,7 @@ async def get_open_orders(self) -> List[OpenOrder]:
             ret_val.append(
                 OpenOrder(
                     client_order_id=client_order_id,
-                    trading_pair=await OBDS.trading_pair_associated_to_exchange_symbol(order["symbol"]),
+                    trading_pair=await AscendExAPIOrderBookDataSource.trading_pair_associated_to_exchange_symbol(order["symbol"]),
```

Comment:
We should be passing the api_factory and the throttler instance in all places where we use the method to translate symbols to trading pairs and trading pairs to symbols